[2022-07-19 10:09:01,206] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T10:08:00+00:00 [queued]>
[2022-07-19 10:09:01,219] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T10:08:00+00:00 [queued]>
[2022-07-19 10:09:01,220] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:09:01,220] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 10:09:01,221] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 10:09:01,240] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): say_hi> on 2022-07-19 10:08:00+00:00
[2022-07-19 10:09:01,246] {standard_task_runner.py:52} INFO - Started process 6888 to run task
[2022-07-19 10:09:01,251] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'say_hi', 'scheduled__2022-07-19T10:08:00+00:00', '--job-id', '344', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpehzbgshe', '--error-file', '/tmp/tmpxwob51dy']
[2022-07-19 10:09:01,252] {standard_task_runner.py:80} INFO - Job 344: Subtask say_hi
[2022-07-19 10:09:01,328] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.say_hi scheduled__2022-07-19T10:08:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 10:09:01,435] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=say_hi
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T10:08:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T10:08:00+00:00
[2022-07-19 10:09:01,437] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-19 10:09:01,438] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Hi!!"']
[2022-07-19 10:09:01,451] {subprocess.py:85} INFO - Output:
[2022-07-19 10:09:01,454] {subprocess.py:92} INFO - Hi!!
[2022-07-19 10:09:01,455] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-19 10:09:01,521] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=say_hi, execution_date=20220719T100800, start_date=20220719T100901, end_date=20220719T100901
[2022-07-19 10:09:01,666] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 10:09:01,765] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
